packages/dds/tree/src/feature-libraries/forestRepairDataStore.ts

```diff
@@ -158,31 +158,29 @@ export class ForestRepairDataStore implements RepairDataStore {
     ): ITreeCursorSynchronous[] {
         const parent = getDescendant(this.root, path);
         const sparseField = parent.children.get(field);
-        assert(sparseField !== undefined, ERR_MISSING_DATA);
+        assert(sparseField !== undefined, "No repair data found");
```

Comment:
@yann-achard-MS FYI, the assert tagger doesn't like this pattern. ☹️  I just removed the constant as a workaround.